components/list/list-item-placement-marker.js

```diff
@@ -0,0 +1,49 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemPlacementMarker extends LitElement {
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+				position: relative;
+			}
+
+			:host([hidden]) {
+				display: none;
+			}
+
+			svg {
+				width: 100%;
+			}
+
+			circle {
+				stroke: var(--d2l-color-celestine);
+				fill: white;
+			}
+
+			line {
+				stroke: var(--d2l-color-celestine);
+				stroke-linecap: round;
+			}
+		`;
+	}
+
+	constructor() {
+		super();
+		this.strokeWidth = 4;
+		this.radius = this.strokeWidth * 2;
```

Comment:
The designs have the radius set to 4 when the stroke width is 3 - I'm unsure if we should be calculating the radius this way or what formula we should use. @m6jones ?
**Current**
![Screen Shot 2020-05-21 at 11 03 20 AM](https://user-images.githubusercontent.com/2412740/82572992-314de200-9b53-11ea-93ba-d44192df13e9.png)
**Design**
![Screen Shot 2020-05-21 at 11 04 04 AM](https://user-images.githubusercontent.com/2412740/82572999-3448d280-9b53-11ea-9977-627142c1621a.png)

Reply:
Yeah, we should be following the designs. Which Abbie just posted!

Reply:
![image](https://user-images.githubusercontent.com/14826663/82575225-2a749e80-9b56-11ea-8d20-1abf1917073e.png)